packages/cli/src/util/createServer.ts

```diff
@@ -97,10 +104,25 @@ function pipeOutputToSignale(stream: Readable) {
   });
 }
 
-export type CreatePrismOptions = {
+function isProxyServerOptions(options: CreateBaseServerOptions): options is CreateProxyServerOptions {
+  return !options.dynamic && 'upstream' in options;
+}
+
+type CreateBaseServerOptions = {
   dynamic: boolean;
   cors: boolean;
-  host?: string;
+  host: string;
   port: number;
-  operations: IHttpOperation[];
+  document: string;
+  multiprocess: boolean;
+  log: 'stdout' | 'httpResponse' | 'httpHeader';
```

Comment:
That's a very good point. I'll change the option type.

Reply:
@philsturgeon

Now that I am reading the proposal, with these options I would be able to understand where the error is going. The only reason I got it is because you wrote it in the parentheses. On this side, having `stdout vs httpHeader vs httpBody/httpResponse` easier to get (I guess).

We can definitely add such information in the CLI documentation, but the parameter does not look that intuitive.

Thoughts?

Reply:
Ok, but some main points:

- no camel case in the CLI
- logging should be default 
- httpResponse is vague we are trying to make it fail for us, maybe —validation enables that, or —error 

Maybe logging and header could always happen and the only switch is —error?

@marbemac need second opinion

Reply:
I'm catching up - by "log" do we mean where to put contract validation errors?

Reply:
@marbemac That is correct!

Reply:
Got it - so yeah for me `log` has a pretty clear existing meaning that !== where contract test results are put. Ideas for a different name? `--report-contract-violations=stdout | header | response`?

Reply:
I am really struggling to find a good name :) — on the other hand I want to make sure we do not spend ages on this…

Reply:
`--report-contract-violations`, `--reporter`, `--report-violations`? 

`log` just has so much baggage, and we'll probably want to reserve that flag for tradditional logs if/when we have that.

Reply:
I'll go with `--report-violations`